### ISSUE TITLE
No test or cross validation set

### ISSUE THREAD

**Issue by JD-P** [JD-P/HNStoryRecommendations#1]
Currently the code does not set aside a test or cross validation set to prevent overfitting
